operator-pipeline-images/operatorcert/entrypoints/check_permissions.py

```diff
@@ -143,9 +143,14 @@ def reviewers(self) -> list[str]:
         Operator reviewers from the operator config file
 
         Returns:
-            list[str]: A list of github users who are reviewers for the operator
+            list[str]: A list of github users who are reviewers for the
+            operator in lowercase format
         """
-        return self.base_repo_operator_config.get("reviewers") or []
+        reviewers = self.base_repo_operator_config.get("reviewers") or []
```

Comment:
I am wondering if maybe we want to also make this change for the property `maintainers`? But It seems that maintainers not being unified to lowercase don't break anything, so this is just a thought.

Reply:
I was initially also thinking about it but then I rejected the idea because we control the list of maintainers (we can edit it anytime) and it is not used the same way as the list of reviewers.

Reply:
Good argument. Feel free to disregard my comment and merge as-is :+1: